editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/restconfiguration/RestConfigEditor.java

```diff
@@ -548,6 +561,35 @@ public void widgetSelected(SelectionEvent e) {
 		
 		return client;
 	}
+	
+	private void addComponentDependency(String componentName) {
+		IWorkbench wb = PlatformUI.getWorkbench();
+		IProgressService ps = wb.getProgressService();
+		try {
+			ps.busyCursorWhile(new IRunnableWithProgress() {
```

Comment:
I think it will be more readable if the inner class is extracted to its own class file